examples/technology_showcase/Rotor_Blade_Inverse_solve.py

```diff
@@ -169,23 +169,21 @@ def display_image(image_name):
 # Define units system and store variables
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 # Select MKS units
-ExtAPI.Application.ActiveUnitSystem = (
-    Ansys.ACT.Interfaces.Common.MechanicalUnitSystem.StandardMKS
-)
+ExtAPI.Application.ActiveUnitSystem = MechanicalUnitSystem.StandardMKS
```

Comment:
Do you keep ExtAPI in this case since you're setting something?

Reply:
Yes.